plugin-dev/Source/Sentry/Private/Desktop/SentrySubsystemDesktop.cpp

```diff
@@ -104,11 +95,24 @@ void SentrySubsystemDesktop::InitWithSettings(const USentrySettings* settings, U
 	}
 
 #if PLATFORM_WINDOWS
-	sentry_options_set_handler_pathw(options, *FPaths::ConvertRelativePathToFull(HandlerPath));
-	sentry_options_set_database_pathw(options, *FPaths::ConvertRelativePathToFull(DatabasePath));
+	if(!FSentryModule::Get().IsMarketplaceVersion())
+	{
+		if(!FPaths::FileExists(GetHandlerPath()))
```

Comment:
This is probably a nitpick but can we keep the path as a local variable so we don't need to `FPaths::Combine` and `ToFullPahth` twice? Here and where we actually set it?

Reply:
I've put back the local variable within the `if(!FSentryModule::Get().IsMarketplaceVersion())` block here to avoid calling those path utils twice.